### DISCUSSION TITLE
[READ THIS FIRST] Welcome to {golem} Discussions!

### DISCUSSION THREAD

**Discussion by ColinFay** [ThinkR-open/golem#567] [General]
## 👋 Welcome!
  
We’re experiencing with using "GitHub Discussions" as a place to connect with other members of the `{golem}` community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about you!

Before opening a new discussion, please check the following:

- Please be aware that the [Code of Conduct](https://github.com/ThinkR-open/golem/blob/master/CODE_OF_CONDUCT.md) also applies to this discussion space. 
- Look for any other discussion on the topic here or in the issues, so that we don't have to treat duplicate discussions. 
- If you open a discussion about a bug or something you haven't understand, please be as precise as possible if you want other to help you. 

Cheers!

**Comment by rpodcast** [comment]
Greetings from a very happy {golem} user 😀.  I'm going to start sharing more of my tips and tricks from my year's journey of releasing 3 major Shiny apps to production that used {golem} but I am curious if there are some lesser known features in the package that people would like to learn more about?

**Reply by rpodcast** [reply]
I was pretty silly, this should have been a dedicated topic in the general category.  I'm still learning this interface!

**Reply by ColinFay** [reply]
Thanks for joining us Eric :)

**Comment by aftonsteps** [comment]
Hi! I'm Afton and I've made Shiny apps for a little while but recently built my first `{golem}` app.

**Reply by ColinFay** [reply]
Welcome here! 

Hope you're enjoying `{golem}` :)

**Comment by jacciz** [comment]
Hello! I've developed a couple Shiny apps as a hobby and just recently (and successfully) converted one into a Golem app. I'm now using Golem for other apps I'm developing.
